src/BizHawk.Client.EmuHawk/config/EmuHawkOptions.cs

```diff
@@ -69,6 +69,7 @@ private void GuiOptions_Load(object sender, EventArgs e)
 			RunInBackgroundCheckbox.Checked = _config.RunInBackground;
 			AcceptBackgroundInputCheckbox.Checked = _config.AcceptBackgroundInput;
 			AcceptBackgroundInputControllerOnlyCheckBox.Checked = _config.AcceptBackgroundInputControllerOnly;
+			DiscordRPC.Checked = _config.DiscordRPC;
```

Comment:
I think I mentioned this in either IRC or Discord when you were there discussing this feature: there's no precedent for this in the codebase (see #2235). I don't think we should hide this setting in `Config` > `Customize...` with the miscellany. It needs its own `TabPage` at the very least, and probably deserves its own dialog with some warning text in bold.